src/app/shared/dialogs/dialogs-resources-viewer.component.ts

```diff
@@ -1,11 +1,32 @@
+import { Router, ActivatedRoute } from '@angular/router';
 import { Component, Inject } from '@angular/core';
-import { MAT_DIALOG_DATA } from '@angular/material';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+
 
 @Component({
-  template: `<planet-resources-viewer [isDialog]="true" [resourceId]="data.resourceId"></planet-resources-viewer>`
+  template: `
+    <div mat-dialog-content>
```

Comment:
The `div` isn't necessary.  You can just make this a `mat-dialog-content` element.  The same goes for the `mat-dialog-actions`